charts/merlin/values.yaml

```diff
@@ -219,6 +202,28 @@ merlin:
     # This should be the actual DNS registered
     trackingURL: "http://www.example.com"
 
+  ui:
```

Comment:
Can you also update https://github.com/gojek/merlin/blob/main/charts/merlin/values-e2e.yaml?

Reply:
`values-e2e.yaml` has no UI config keys

Reply:
Ah yeah